src/cmdline/mod.rs

```diff
@@ -72,18 +61,6 @@ fn valid_str(s: &str) -> Result<()> {
     }
 }
 
-fn valid_element(s: &str) -> Result<()> {
```

Comment:
Why has this been removed? Components of the boot args (keys, corresponding values and standalone boot args) should still be validated.

Reply:
it was removed because this method was used only inside the methods removed (and mentioned above). Considering the removal of these methods the valid_element was no longer needed. Considering the comments above, I'll rollback this method as well.